react/ListPrice.tsx

```diff
@@ -19,8 +24,10 @@ const ListPrice: StorefrontFC<BasicPriceProps> = props => {
     return null
   }
 
-  const listPriceValue = commercialOffer.ListPrice
+  const listPriceValue: number = commercialOffer.ListPrice
```

Comment:
Is this `: number` needed? What's the type of the `commercialOffer.ListPrice`

Reply:
I had to do this to solve a lint problem D:
I will try to type these things correctly later